crates/pipeline-manager/tests/integration_test.rs

```diff
@@ -1950,6 +1950,7 @@ async fn pipeline_logs() {
             "program_code": "CREATE TABLE t1(c1 INTEGER) with ('materialized' = 'true');"
         }))
         .await;
+    sleep(Duration::from_millis(100)).await; // Wait briefly for the pipeline runner to start
```

Comment:
usually these things end up causing us pain by being flaky in the test harness :) - maybe there is a better way

Reply:
So far we have not had any issue with it (this condition has been in the code for a couple of months), I added it because I initially didn't have the notifier preempting, so the runner starting every second was sufficient time for the logs call to still fail as the runner was not yet started along to be able to provide the log.

I can add it trying for 4s regularly before giving up, that should make it not flaky.